swagger.yaml

```diff
@@ -2370,6 +2370,9 @@ components:
   schemas:
     BillableMetricGroup:
       type: object
+      required:
+        - key
+        - values
```

Comment:
BillableMetricGroup is also used for update action so I guess we don't need require restrictions here

Reply:
Thanks for the explanation. Let me go once again through all changes and I think that we are ready then for the merge!